tpch/run-tpch-dbgen.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+cd /tpch-dbgen
+./dbgen -vf -s 1
```

Comment:
Maybe we can allow user to specify scale factor?

Reply:
yes👍